tests/collectors/test_ndbc_sos.py

```diff
@@ -34,13 +37,13 @@ def test_raw_ndbc_get_observation(self):
         self.c.features     = ['41012']
         self.c.variables    = ['air_pressure_at_sea_level']
 
-        response = self.c.raw(responseFormat="text/csv")
-        assert isinstance(response, basestring)
+        response = self.c.raw(responseFormat="text/csv").decode()
```

Comment:
I didn't realize this, but decode defaults to utf-8, that's really good to know for me.